src/styles/blocks/card.css

```diff
@@ -0,0 +1,88 @@
+.card {
+  background-color: #fff;
+  color: var(--text-color-main-accent);
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid var(--card-border-color);
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  align-self: center;
+  margin-bottom: 40px;
+}
+
+.card__title {
+  margin: 0;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin-bottom: 4px;
+}
+
+.card__info,
+.card__price-text {
+  color: var(--text-color-secondary);
+}
+
+.card__info {
+  margin-bottom: 16px;
+}
+
+.card__rating-info {
+  align-self: end;
+}
+
+.card__rating-container {
+  margin-bottom: 26px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__price-text {
+  font-size: 12px;
+}
+
+.card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+}
+
+.card__price-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 18px;
+}
+
+.card__info,
+.card__rating-info {
+  font-size: 10px;
+}
+
+.card__button {
+  height: 40px;
+  width: 100%;
+  background-color: var(--card-button-color);
+  color: var(--text-color-button);
+  font-size: 14px;
+  border-radius: 5px;
+  text-decoration: none;
+  text-transform: uppercase;
+  display: flex;
+  text-align: center;
+  align-items: center;
+  justify-content: center;
+  border: 1px solid transparent;
+}
```

Comment:
try to follow CSS properties order (https://9elements.com/css-rule-order/)

Reply:
Oh! Thank you for the helpful info!